src/webextension/telemetry.js

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export async function recordEvent(category, method, object, value, extra) {
```

Comment:
There's a few pieces of common data we'll need to gather for just about every event:

* FxA user identifier, when available (gotten from `Authorization.uid`)
* Item identifier, when available (gotten from `item.id`)
* action/flow identifier (preferably a UUID so it has a good chance of being unique across runs, views)